Export-Excel.ps1

```diff
@@ -620,27 +623,36 @@
           if it is a data table don't do foreach on it (slow) - put the whole table in and set dates on date columns,
           set things up for the end block, and skip the process block #>
         if ($InputObject -is  [System.Data.DataTable])  {
-            #Change TableName if $TableName is non-empty; don't leave caller with a renamed table!
-            $orginalTableName = $InputObject.TableName
-            if ($TableName) {
-                $InputObject.TableName = $TableName
-            }
-            while ($InputObject.TableName -in $pkg.Workbook.Worksheets.Tables.name) {
-                Write-Warning "Table name $($InputObject.TableName) is not unique, adding '_' to it "
-                $InputObject.TableName += "_"
-            }
-            #Insert as a table, if Tablestyle didn't arrive as a default, or $TableName non-null - even if empty
-            if ($null -ne $TableName -or $PSBoundParameters.ContainsKey("TableStyle")) {
-                $null = $ws.Cells[$row,$StartColumn].LoadFromDataTable($InputObject, (-not $noHeader),$TableStyle )
-                # Workaround for EPPlus not marking the empty row on an empty table as dummy row.
-                if ($InputObject.Rows.Count -eq 0) {
-                    ($ws.Tables | Select-Object -Last 1).TableXml.table.SetAttribute('insertRow', 1)
+            if ($Append) {
```

Comment:
Maybe do `if ($Append -and $ws.Dimension)` like in line 553 (Or something better if you think of it)
So when you do something like in https://github.com/dfinke/ImportExcel/issues/730#issue-525769283 example:
```powershell
for ($i = 0; $i -lt 3; $i++) {
    Export-Excel -InputObject $DataTable -Path $Path -TableName 'Test2' -Append
}
```
The first -Append will be ignored